notes.txt

```diff
@@ -24,3 +24,20 @@ buffer down to 100k
 
 1. v360ish -- entered experimental mode; freely adjusted learning rate up and
 down, adjusted batch size, etc, to see if valnet divergence could be fixed.
+
+
+== v7
+
+Started training with model 174.  Moved minimum games before starting training
```

Comment:
Rolled back to v5 model 173 and continue from there with first v7 model being 174

Reply:
Done